src/printer.js

```diff
@@ -607,9 +614,8 @@ function genericPrintNoParens(path, options, print) {
         path.call(print, "pattern")
       ]);
     // Babel 6
-    case "ObjectProperty":
-    case // Non-standard AST node type.
-    "Property":
+    case "ObjectProperty": // Non-standard AST node type.
```

Comment:
Those comments are now shifted by one due to different buggy versions of our comment handling algorithm :p